docs/data/material/migration/migration-grid-v2/migration-grid-v2.md

```diff
@@ -0,0 +1,80 @@
+# Migration to Grid v2
+
+<p class="description">This guide explains how and why to migrate from Material UI Grid v1 to v2.</p>
+
+## Why you should migrate
+
+The Grid v2 has new feature and a lot of improvements:
+
+- Easier to customize because Grid v2 uses CSS variables which remove CSS specificity from class selector. Now you can use `sx` prop on the Grid to control any style you'd like.
+- All grids are considered items without specifying the `item` prop.
+- The Long-awaited [offset feature](/material-ui/react-grid2/#offset) is added.
+- [Nested grid](/material-ui/react-grid2/#nested-grid) has no depth limitation.
+- The `disableEqualOverflow` flag is added to prevent scrollbar in small viewports.
+
+:::info
+We ship the Grid v2 as `Unstable_` to let the community try it out and collect feedbacks. We will make it stable and deprecate v1 in the next major release of Material UI.
+:::
+
+## Material UI v4
+
+The Grid v2 is introduced in Material UI v5, so you have to follow the [Material UI migration guide](/material-ui/migration/migration-v4/) first.
+
+## Material UI v5
+
+The migration is expected to be smooth since most of the APIs remains the same. However, there is one breaking change that we want to clarify:
```

Comment:
This line mentions that it will clarify the one breaking change, but after reading this section it's not clear to me what that is. 😅

Reply:
I see. The breaking change is that Grid v2 reverted the negative margin approach back to what Material UI v4 has.

It sounds confusing, so maybe this help explaining it:

- Material UI v4: the Grid has negative margin on all sides
- Material UI v5: the Grid is updated to have negative margin on top & left. However, it comes with a cost which does not worth the change.
  - The Grid v2 is created to have negative margin on all sides by default.

Reply:
Is removing the `:::warning` enough?

Reply:
Ohhh I see, yeah, the callout makes it feel like it's separate from the rest of the text. I think it's more clear if we remove the :::warning.